docker/Dockerfile.dev

```diff
@@ -6,12 +6,17 @@ ENV PATH "/root/.poetry/bin:$PATH"
 
 WORKDIR /srv/app
 
-RUN apk add --no-cache \
+RUN rm -rf /var/cache/apk/* && \
+    rm -rf /tmp/* && \
+    apk update && \
```

Comment:
seems more robust to update the listing, good change. Usually images come with an empty package listing, so you don't have to remove things. An explicit `apk update` may or may not be necessary, but it's good form to make it explicit.